tests/examples/hello/src/iota.cc

```diff
@@ -0,0 +1,28 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class IotaTask : public Task<IotaTask, IOTA> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &output = context.outputs()[0];
+    auto output_shape = output.shape<1>();
```

Comment:
I suggest adding assertions here (and maybe explain in a comment), to show that the dimensions and type of a Store are dynamic, and what we "receive" on the task side (in C++) depends on what the calling code (in python) passes in. Here the rest of the task can only work on 1d float arrays, so we check that the calling code indeed passed one such array.

```
assert(output.code() == FLOAT_LT);
assert(output.dim() == 1);
```

Reply:
👍